lib/spaceship/tunes/app_analytics.rb

```diff
@@ -0,0 +1,145 @@
+module Spaceship
+  module Tunes
+    class AppAnalytics < TunesBase
+      # @return (Integer) The app ID to grab analytics for
+      attr_accessor :apple_id
+
+      attr_mapping({
+        'adamId' => :apple_id
+      })
+
+      class << self
+        def factory(attrs)
+          return self.new(attrs)
+        end
+      end
+
+      def app_units
```

Comment:
I didn't even know that existed in ruby. Yeah that's way better. I can take a stab at adding that.